### DISCUSSION TITLE
news/post/dac588d9-f575-4eef-b830-f21f1c3fb35d

### DISCUSSION THREAD

**Discussion by giscus** [SimYunSup/newsletter-ones-to-watch#1] [General]
# news/post/dac588d9-f575-4eef-b830-f21f1c3fb35d

Astro description

https://ones-to-watch.ethansup.net/news/post/dac588d9-f575-4eef-b830-f21f1c3fb35d

**Comment by SimYunSup** [comment]
여러분을 응원합니다 :smile:

**Comment by pumpkiinbell** [comment]
좋은 글 공유 감사합니다~

**Comment by ChoiWonBeen** [comment]
요번주는 참여를 못했네요 ㅠ 다음주에 참여하겠습니다 :fire:
